src/features/Roulette/components/RouletteFilter/RouletteFilter.module.scss

```
@@ -51,31 +53,31 @@
-        margin-right: 6px;
-      }
+    @include m.mq(lg) {
+      @include m.text-style(md, 400);
+      height: 40px;
+      padding: 0 18px;
     }
   }
+
+  &__options__item__icon {
+    @include m.flex-box(center, center);
+    margin-right: 6px;
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)